src/edu/csus/ecs/pc2/exports/ccs/ResultsFile.java

```diff
@@ -96,16 +96,12 @@ private int getMedian(StandingsRecord[] srArray) {
      * @return
      * @throws IllegalContestState
      */
-    public String[] createTSVFileLines(IInternalContest contest, String resultFileTitleFieldName) throws IllegalContestState {
+    public String[] createTSVFileLines(IInternalContest contest, String resultFileTitleFieldName)  {
 
         Vector<String> lines = new Vector<String>();
 
         finalizeData = contest.getFinalizeData();
         
-        if (finalizeData == null){
-            throw new IllegalContestState("Contest not Finalized, can not create results file");
-        }
-
         NewScoringAlgorithm scoringAlgorithm = new NewScoringAlgorithm();
```

Comment:
I _really_ question why this code (or any code) is using `NewScoringAlgorithm`, since I thought we agreed that all scoring computations should be done using the `DefaultScoringAlgorithm` class.  Further, there's not much help provided by looking at the `NewScoringAlgorithm` class, since all its description says is 
```
 * "New" Scoring Algorithm implementation.
 * 
 * Uses same SA as the {@link DefaultScoringAlgorithm}
 * 
```
and yet the code in `NewScoringAlgorithm` is radically different (to the point that a Compare With Each Other fails to be able to identify corresponding sections.)

But since the above isn't part of this PR, I guess that's an issue for another time...

Reply:
Consider either referencing an existing Issue or using the ... to create an issue, rather than just ignore this.

Reply:
Done; I've submitted a new Issue on this:  #846.  Marking this particular conversation "Resolved".